shared/src/business/entities/DocketEntry.test.js

```diff
@@ -1943,4 +1944,50 @@ describe('DocketEntry entity', () => {
       expect(docketEntry.isValid()).toBeTruthy();
     });
   });
+
+  describe('generateFiledBy', () => {
```

Comment:
There's already a describe block in this test file that's called "generate filed by string" testing other filedBy  scenarios - it seems like these new tests could be organized along with those.

Reply:
Good call, update coming!

Reply:
Updated!